.github/workflows/quicktest.yml

```diff
@@ -0,0 +1,79 @@
+name: Quick Tests
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - develop
+
+jobs:
+  # Performs some quick tests.
+  # This is a much quicker test suite which only runs the unit tests and linting
+  # against the low/high supported PHP/PHPCS combinations.
+  #
+  # Performs the following steps:
+  # - Checks out the repository.
+  # - Configures caching for Composer.
+  # - Sets up PHP.
+  # - Installs PHPCS.
+  # - Install Composer dependencies.
+  # - Runs the Composer linting script (if desired).
+  # - Runs Composer run-tests script.
+  quick-tests:
+    name: PHP ${{ matrix.php }} on PHPCS ${{ matrix.phpcs_branch }}${{ matrix.lint && ' with linting' || '' }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '7.4', '5.4' ]
+        phpcs_branch: [ 'dev-master' ]
+        include:
+          - php: '7.3'
+            phpcs_branch: '3.5.0'
+          - php: '5.4'
+            phpcs_branch: '3.5.0'
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Set up Composer caching
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-composer-dependencies
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        if: ${{ matrix.phpcs_branch == 'dev-master' }}
+        with:
+          php-version: ${{ matrix.php }}
```

Comment:
I looked into this a little bit. It seems based on [this](https://github.com/shivammathur/setup-php/blob/7b44f7e007410d13d3ed2deec1d452256eb79d96/src/install.ts#L27) and [this](https://github.com/shivammathur/setup-php/blob/7b44f7e007410d13d3ed2deec1d452256eb79d96/src/utils.ts#L20-L42) that an empty string would be returned if the input is not required, and a value is not supplied.

But it's probably better for clarity to just define `none`. It will make it more clear for someone unfamiliar with `setup-php`.